governance-charter.md

```diff
@@ -159,19 +164,18 @@ to the respective positions.
 
 ### Maximal representation
 
-To encourage diversity there will be a maximum of one-third representation on
-the Governance Committee from any one company at any time. If the outcomes of
-an election result in greater than 1/3 representation (or maximum of two,
-whichever is greater), the lowest vote getters from any particular company will
-be removed until representation on the committee is less than one-third.
-
-If percentages shift because of job changes, acquisitions, or other events,
-sufficient members of the committee must resign until max one-third
-representation is achieved. If it is impossible to find sufficient members to
-resign, the entire company’s representation will be removed and new special
-elections held. In the event of a question of company membership (for example
-evaluating independence of corporate subsidiaries) a majority of all
-non-involved Governance Committee members will decide. 
+To encourage diversity there will be a maximum of two Governance Committee
+members from any one company at any time. If the outcomes of an election result
+in more than two candidates from the same company being elected, the lowest vote
+getters from any particular company will be removed.
+
+If representation changes because of job changes, acquisitions, or other events,
+sufficient members of the committee must resign until the maximum representation
+is respected. If it is impossible to find sufficient members to resign, the
+entire company’s representation will be removed and new special elections held.
```

Comment:
```suggestion
If representation changes because of job changes, acquisitions, or other events,
sufficient members of the committee must resign until the maximum representation
is respected. If it is impossible to find sufficient members to resign, the
entire company’s representation will be removed. In both cases a special election will be held for those position, 
as [outlined below](#special-elections)
```

The current wording creates the assumption that special elections are only held, if the members are removed (forcefully), but reading the section about Special Elections below, this is also the case when members resign voluntarily.